### PR TITLE
Migrating from PTL notebook

### PR DESCRIPTION
**Pull request by florescl**

### CODE REVIEW

**Comment by hanlint** [comment]
This looks pretty good! I will do an editing pass and merge this.

**Comment by moinnadeem** [comment]
What is the status of this PR? I'm adding notebook links to our README

**Comment by florescl** [comment]
@hanlint is reviewing asap

**Comment by hanlint** [comment]
Apologies for the delay, will edit and merge this tomorrow.

**Review by hanlint — COMMENTED** [review]
I updated the text/code slightly, but having some trouble getting the PTL portion to work -- @florescl could you help here?

**Comment by moinnadeem** [comment]
Is this PR close to being merged by Friday? I'm updating the README. I think it would be great to include before we go pub-ready.

**Comment by florescl** [comment]
@hanlint this is ready, can you approve?

File: notebooks/Migrating_from_PTL.ipynb [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install mosaicml \n",
+    "!pip install pytorch-lightning\n",
+    "!pip install git+https://github.com/mosaicml/composer.git@dev"
```
Comment (hanlint) [inline notebooks/Migrating_from_PTL.ipynb:27]:
```suggestion
```

**Review by hanlint — APPROVED** [review]

**Pull request merged**